### ISSUE TITLE
Project closure

### ISSUE THREAD

**Issue by davemckain** [davemckain/qtiworks#82]
I've decided to finally close this old project - unfortunately I no longer have the time or resources to give this the support it needs.

I'd be happy to transfer ownership over to any individual or organisation who wishes to continue maintaining it.

I'm going to shut down the QTIWorks demo instance on **Friday 31st March** and delete all user data from the system. I've emailed all users with individual accounts informing them of this; all LTI domains had already been disabled.

**Comment by foersant** [comment]
I have no idea, if I can get this done - but please let's get in touch. I wrote you an email.

**Comment by davemckain** [comment]
The QTIWorks demo instance has now been deleted.

I made a final backup of remaining user data (e.g. uploaded assessments, results etc.) just in case I get any late date requests from former users. I will retain this backup for 3 months before deleting it permanently.

**Comment by davemckain** [comment]
I've now deleted the final user data I had extracted and backed up when shutting down the University of Edinburgh QTIWorks instance on 17th April.

There is now no longer any QTIWorks user data left on hosts at the University of Edinburgh. 

However the just-deleted data still exists on local tape backup services, where it will continue to be retrievable for ~60 days before being written over. I will let this automatic ageing process continue as normal unless someone explicitly requests otherwise.
